e2e/e2e_test.go

```diff
@@ -10,7 +10,8 @@ import (
 
 	_ "github.com/hashicorp/nomad/e2e/affinities"
 	_ "github.com/hashicorp/nomad/e2e/clientstate"
-	_ "github.com/hashicorp/nomad/e2e/connect"
+
+	// _ "github.com/hashicorp/nomad/e2e/connect"
```

Comment:
If I recall correctly the restarts are because of Consul ACLs. Is there a subset of tests we could run that don't require the ACLs setup, so that we can at least get some coverage here?

Reply:
The ACL ones were particularly bad but the others also had some bad flakiness. I'll do some basic analysis now and re-activate in a follow up PR and open an issue with notes soon as well.